assets/dev/scss/frontend/breakpoints/breakpoints.scss

```diff
@@ -1,11 +1,11 @@
 $screen-xs-max: map_get($breakpoints, sm) - 1;
```

Comment:
Remove this variable and use a hardcoded value when it's needed

Reply:
Done.